src/collar/oc_core.lsl

```diff
@@ -46,11 +46,6 @@ integer g_iVerbosityLevel=1;
 
 integer g_iNotifyInfo=FALSE;
 
-/*string MajorMinor(){
```

Comment:
In order to keep the scripts clean over time for everyone, let's prefer deletion over commenting things out like this.

Reply:
My inclination is to leave a deleted function commented out for one revision for the sake of easily seeing what's been changed, but I'll keep this in mind in future. In this case as the commented functions have just been inlined into the body of the code to save memory it's not really an issue anyway.